databases/schemas/roomlogs.sql

```diff
@@ -0,0 +1,16 @@
+CREATE TABLE public.roomlogs (
+	type STRING NOT NULL,
+	room STRING NOT NULL,
+	userid STRING NULL,
+	day STRING NOT NULL,
+	month STRING NOT NULL,
+	year STRING NOT NULL,
+	timestamp INT NOT NULL,
+	log STRING NOT NULL,
+	INDEX userid (userid),
+	INDEX room (room),
+	INDEX type (type),
+	INDEX fulldate (day, month, year)
```

Comment:
These indexes don't make sense to me. I don't think you'll ever want to pull unsorted logs associated with users or rooms or types?

`INDEX userid (userid, time)` and `INDEX room (room, time)` make sense to me.